03/app.js

```diff
@@ -42,10 +42,33 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
-
-}
+    
+        const plTitle = this[isbn]['title']['pl']
+        const enTitle = this[isbn]['title']['en']
```

Comment:
ok, chociaż lepszym i bardziej elastycznym rozwiązaniem byłoby `return this[isbn]['title'][lang]`